.deepsource.toml

```diff
@@ -0,0 +1,31 @@
+version = 1
+
+exclude_patterns = [
+    "public/**,",
+    "build/**",
+    "node_modules/**"
+]
+
+[[analyzers]]
+name = "docker"
+enabled = true
+
+[[analyzers]]
+name = "secrets"
+enabled = true
+
+[[analyzers]]
+name = "javascript"
+enabled = true
+
+[analyzers.meta]
+module_system = "es-modules"
+environment = [
+    "nodejs",
+]
+dialect = "typescript"
+skip_doc_coverage = ["class-expression", "method-definition"]
```

Comment:
I think its a valid `toml` syntax but would be more readable if this sub "block" is indented?